sns/cm/repair/ut/xform.c

```diff
@@ -438,6 +438,87 @@ static void cp_multi_failures_post(char data, int cnt, int index)
 	m0_semaphore_down(&sem);
 }
 
+static void buf_initialize(struct m0_buf *buf, uint32_t size, uint32_t len)
```

Comment:
to say that this is an array of pointer to "struct m0_buf"
```
static void buf_initialize(struct m0_buf *buf_arr, uint32_t arr_size, uint32_t len)
```

Or at least 
```
static void buf_initialize(struct m0_buf *bufs, uint32_t size, uint32_t len)
```

Reply:
Done.